data/human/human missions.txt

```diff
@@ -3996,3 +3996,91 @@ mission "Fumbling Mugger"
+			choice
+				`	"Captain <last>, at your service. We leave at dawn!"`
+				`	(Stay silent)."`
+			`	You wake up significantly after dawn with empty bottles and unconscious men strewn all over the ship's mess hall. Karengo peels himself off the floor and gives you a crooked smile, shielding his eyes from the light. "Alright, <last>" he says, "let's do this."`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying the passengers hasn't entered this system yet. Better depart and wait for it.`
+	on stopover
+		conversation
+			`	The men load into a shuttle and Karengo directs you down through the atmosphere of <planet>, steering clear of the usual tourist areas and, you realize, of the local law enforcement. As you're putting down in a clearing halfway up a mountain, you hear an otherworldly screech and your ship lurches violently to one side. You would have been thrown to the floor if you hadn't been strapped in.`
+			`	"Look lively, lads!" shouts Karengo, the only person still standing. "Your chariot awaits!" He double-checks the jetpacks on the backs of each of the men, or "Dragonriders", as they creatively branded themselves during their first night of the voyage, then opens the side hatch and hurls the Dragonriders out the door before jumping himself.`
+			`	After the group has left, you land - this time undisturbed - and pull up every external camera you have to watch the action.`
+			`	The dragons of <planet> you learned about on the "Xenofauna and Flora" channel were different from the dragons of mythology, but the beasts you see today aren't quite what you learned about, either. For one, the creatures flying outside your window are largely invisible - a trick of the way the light reflects around the scales on their hides. They are faster, too, or perhaps the ones on the show had been drugged to make them easier to film.`
+			`	A quick survey of the scene shows that Karengo has tagged three dragons with some sort of harmless marker gun to make them easier to track. Eight of the Dragonriders are chasing the largest one, which is missing part of a tooth and has blackened scales from where it hit your shuttle's shield. The last Dragonrider is nowhere in sight.`
+			`	Suddenly, proximity sensors go off on the starboard side and a camera shows something - or someone - bounce off your shield and tumble down the mountain.`
+			choice
+				`	(Announce that I'm leaving).`
+					goto leave
+				`	(Stay and watch the Dragonriders).`
+					goto watch
+				`	(Take off and leave everybody else behind).`
+					goto betrayal
```

Comment:
```suggestion
```

Reply:
Why should this line be deleted?

Reply:
Huh, I put this suggestion on the `goto watch` line, and the next one on the `label watch` line, because the watch choice will go straight into that, but these and at least one other comment have apparently been assigned to different lines.

I wonder if this might be related to changes being made in the time between I started the review and ended it.

Reply:
Sorry, that might've been it. To be honest I didn't expect to get a review so soon.